### PR TITLE
DagCBOR: Make Multibase encoding clearer

### PR DESCRIPTION
**Pull request by vmx**
Addressing [@rvagg comment](https://github.com/ipld/specs/pull/163#discussion_r313822792) about more clarity in regards to the Multibase encoding.

### CODE REVIEW

**Comment by vmx** [comment]
I force-pushed a new version. After reading the CID spec I understood what @rvagg meant. The CID spec says that the Multibase may be omitted when encoded as binary. But for CBOR it may not be omitted. I hope things are clearer now. If anyone has a better phrasing of this, please comment.

File: block-layer/codecs/dag-cbor.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ with the tag 42 must be a valid CID.
 ## Link Format
 
 As with all IPLD formats, DagCBOR must be able to encode [Links].
-In DagCBOR, links are encoded using the raw-binary identity [Multibase] in a
-field with a byte-string type (major type 2), with the tag 42.
+In DagCBOR, links are [CIDs] encoded using the raw-binary identity [Multibase]. That Multibase prefix (`0x00`) *must not* be omitted. They are stored as byte-string type (major type 2), with the tag 42.
```
Comment (Stebalien) [inline block-layer/codecs/dag-cbor.md:18]: And it MUST be 0x00 (the identity/base256 multibase multibase).

Comment (vmx) [reply block-layer/codecs/dag-cbor.md:18]: @Stebalien Should I rephrase it that it must be 0x00?

Comment (Stebalien) [reply block-layer/codecs/dag-cbor.md:18]: Nevermind. I misread it the first time.

**Review by Stebalien — APPROVED** [review]

**Review by rvagg — APPROVED** [review]

**Pull request merged**